Cargo.lock

```diff
@@ -5043,8 +5049,13 @@ dependencies = [
  "arrayvec 0.7.1",
  "atomic_refcell",
  "backtrace",
+<<<<<<< Updated upstream
  "base64 0.21.2",
  "bitflags",
+=======
+ "base64 0.10.1",
+ "bitflags 1.3.2",
+>>>>>>> Stashed changes
  "bluetooth_traits",
```

Comment:
This merge conflict needs to be resolved. Also https://github.com/servo/servo/blob/master/servo-tidy.toml#L25 will need a new entry for `bitflags`.

Reply:
Oh sorry I missed when switching the branch. I force pushed fixing this.

Reply:
wait i missed an another line

Reply:
fixed